config/crd/bases/druid.gardener.cloud_etcds.yaml

```diff
@@ -4,7 +4,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
```

Comment:
Something went wrong with the CRD generation because I don't see the new field here. Can you run `make install-requirements` and `make manifests` again?

Reply:
Ha, good catch, added. Dunno why this was not added in the first run.